package.json

```diff
@@ -17,12 +17,14 @@
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
         "clean": "ts-node ./script/clean.ts",
-        "compile": "tsc --build --verbose && npm run postcompile --workspaces --if-present",
```

Comment:
what about calling `postcompile` in the new version?

Reply:
`post` scripts are called by npm itself in nested workspaces correctly. It's only `precompile` that was causing problem.